ffi/kotlin_experimental/pom.xml

```diff
@@ -0,0 +1,158 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.unicode</groupId>
+  <artifactId>icu4x</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>icu4x</name>
+  <url>http://maven.apache.org</url>
+
+    <properties>
+        <kotlin.version>2.1.0</kotlin.version>
```

Comment:
Add
```xml
<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Otherwise some operations will be done with the system encoding, making the build platform dependent.